packages/office-ui-fabric-react/src/components/Nav/Nav.scss

```diff
@@ -103,13 +103,16 @@ $linkRightPadding: 20px;
   display: block;
   position: relative;
 
+  & button {
```

Comment:
please remove this rule. It is a huge assumption to assume all buttons in this element should fill.